### PR TITLE
Renamed images and updated links to images

### PR DESCRIPTION
**Pull request by yash2002109**
While rechecking the repository to ensure it follows the template, in reference to issue #102. I found some images which didn't follow the naming pattern. Fixed them and updated their links in markdown.

Signed-off-by: Yash Prakash <yash2002109@gmail.com>

### CODE REVIEW

File: focus-areas/when/burstiness.md [hunk 1]
```diff
@@ -45,12 +45,12 @@ There are a number of reasons that may prompt a sudden increase or decrease in t
 
 Augur:
 
-![Augur Burstiness](https://raw.githubusercontent.com/chaoss/wg-common/master/focus-areas/when/images/burstiness.augur.png)
+![Augur Burstiness](https://raw.githubusercontent.com/chaoss/wg-common/master/focus-areas/when/images/burstiness_augur.png)
```
Comment (GeorgLink) [inline focus-areas/when/burstiness.md:48]:
```suggestion
![Augur Burstiness](images/burstiness_augur.png)
```

File: focus-areas/when/burstiness.md [hunk 2]
```diff
@@ -45,12 +45,12 @@ There are a number of reasons that may prompt a sudden increase or decrease in t
...
 
 
 GrimoireLab:
 
-![GrimoireLab Burstiness](https://raw.githubusercontent.com/chaoss/wg-common/master/focus-areas/when/images/burstiness.gl.png)
+![GrimoireLab Burstiness](https://raw.githubusercontent.com/chaoss/wg-common/master/focus-areas/when/images/burstiness_gl.png)
```
Comment (GeorgLink) [inline focus-areas/when/burstiness.md:53]:
```suggestion
![GrimoireLab Burstiness](images/burstiness_gl.png)
```

File: focus-areas/when/time-to-first-response.md [hunk 3]
```diff
@@ -26,11 +26,11 @@ Time to first response of an activity = time first response was posted to the ac
 
 
 ### Visualizations
-![GrimoireLab Panel: Efficiency Timing Overview](https://github.com/chaoss/wg-common/blob/master/focus-areas/when/images/efficiency_timing_overview.png)
+![GrimoireLab Panel: Efficiency Timing Overview](https://github.com/chaoss/wg-common/blob/master/focus-areas/when/images/time-to-first-response_efficiency-timing-overview.png)
```
Comment (GeorgLink) [inline focus-areas/when/time-to-first-response.md:29]:
```suggestion
![GrimoireLab Panel: Efficiency Timing Overview](images/time-to-first-response_efficiency-timing-overview.png)
```

File: focus-areas/when/time-to-first-response.md [hunk 4]
```diff
@@ -26,11 +26,11 @@ Time to first response of an activity = time first response was posted to the ac
...
 ---------
-![Augur Visualization: Time to First Response Heat Map ](https://github.com/chaoss/wg-common/blob/master/focus-areas/when/images/augur-ttc-1.png)
+![Augur Visualization: Time to First Response Heat Map ](https://github.com/chaoss/wg-common/blob/master/focus-areas/when/images/time-to-first-response_augur-ttc-1.png)
```
Comment (GeorgLink) [inline focus-areas/when/time-to-first-response.md:31]:
```suggestion
![Augur Visualization: Time to First Response Heat Map ](images/time-to-first-response_augur-ttc-1.png)
```

File: focus-areas/when/time-to-first-response.md [hunk 5]
```diff
@@ -26,11 +26,11 @@ Time to first response of an activity = time first response was posted to the ac
...
 ---------
-![Augur Visualization: Mean Response Times](https://github.com/chaoss/wg-common/blob/master/focus-areas/when/images/augur-ttc-2.png)
+![Augur Visualization: Mean Response Times](https://github.com/chaoss/wg-common/blob/master/focus-areas/when/images/time-to-first-response_augur-ttc-2.png)
```
Comment (GeorgLink) [inline focus-areas/when/time-to-first-response.md:33]:
```suggestion
![Augur Visualization: Mean Response Times](images/time-to-first-response_augur-ttc-2.png)
```

File: focus-areas/when/activity-dates-and-times.md [hunk 6]
```diff
@@ -24,10 +24,10 @@ Individuals engage in activities in open source projects at various times of the
...
 ### Visualizations
 
-<img src="images/1.png" alt="Date_Time_Chart_1" width="700"/>
-<img src="images/2.png" alt="Date_Time_Chart_2" width="700"/>
-<img src="images/3.png" alt="Date_Time_Chart_3" width="700"/>
-<img src="images/4.png" alt="Date_Time_Chart_4" width="700"/>
+<img src="images/activity-dates-and-times_1.png" alt="Date_Time_Chart_1" width="700"/>
+<img src="images/activity-dates-and-times_2.png" alt="Date_Time_Chart_2" width="700"/>
+<img src="images/activity-dates-and-times_3.png" alt="Date_Time_Chart_3" width="700"/>
+<img src="images/activity-dates-and-times_4.png" alt="Date_Time_Chart_4" width="700"/>
```
Comment (GeorgLink) [inline focus-areas/when/activity-dates-and-times.md:30]:
```suggestion
![date time chart 1](images/activity-dates-and-times_1.png)
![date time chart 2](images/activity-dates-and-times_2.png)
![date time chart 3](images/activity-dates-and-times_3.png)
![date time chart 4](images/activity-dates-and-times_4.png)
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
1. change to relative paths
2. change HTML `` tags to Markdown

**Comment by yash2002109** [comment]
Hi @GeorgLink, 
I made the changes as suggested. Please review.

**Pull request merged**